packages/gill/src/__typetests__/transfer-tokens-transaction.ts

```diff
@@ -85,7 +84,6 @@ async () => {
       // @ts-expect-error Should not have a Lifetime
     })) satisfies TransactionMessageWithBlockhashLifetime;
 
-    // @ts-expect-error Should not be a signable transaction
```

Comment:
Worth making the same change as [create-token-transaction.ts](https://github.com/gillsdk/gill/pull/328/files#diff-83499557f20660707558799f914ca9d4adccbb5d83582c59def7b97fd1076059)?